### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by stodge** [piccolo2d/piccolo2d.java#278]
I assume this project is dead and shouldn't be used? It's a shame as it might fit my needs. Is anyone still using it?

**Comment by stodge** [comment]
She's dead, Jim.

**Comment by heuermh** [comment]
I disagree.  There would be some meta-issues to take care of but new releases could be made.
